### DISCUSSION TITLE
Flask學習

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#226] [General]
最近從0開始學習flask ，查遍了很多教學文章發現這個網站寫的不錯，如果大家有什麼相關學習資訊可以分享給我這個廢廢初學者～[https://www.1ju.org/flask/flask-sending-form-data-to-template]()

**Comment by Jie2222** [comment]
感謝同學分享，不過你的連結點進去是同樣這個github的頁面，要複製貼上到網頁上才看的到你分享的網站，不知為啥?

**Reply by enoch-lightning** [reply]
我想可能是因為在Github中貼上URL(Uniform Resource Locator)，也就是「網域名稱(Domain name)」，也就是「網站地址」，也就是「網址」如`https://www.1ju.org/flask/flask-sending-form-data-to-template`的時候，他寫成`[https://www.1ju.org/flask/flask-sending-form-data-to-template](https://github.com/1101-GS4719-C/Python-Community-GS4719-C/discussions/226)`，就變成顯示出來的文字確實是網址，但點產生的超連結，卻是連到目前這個Github(寫github也可以)頁面。

**Comment by jcpan821** [comment]
感謝分享~~
